src/ex2/modules/ItemManager.js

```diff
@@ -0,0 +1,30 @@
+import { PokemonClient } from './PokemonClient.js';
+
+export class ItemManager {
+    constructor() {
+      this.todosArr = [];
+    }
+
+
+    async addPokemons(allIds) {
+    const pokemonsArray = allIds.split(',');
+    const pokemonClient = new PokemonClient();
+    let data;
+    data = await pokemonClient.fetchPokemon(pokemonsArray);
+    data.forEach((pokemon) => {
+      this.todosArr.push('Catch ' + pokemon.name);
+    });
+  }
+
+  addTask(item) {
+    this.todosArr.push(item);
+    return this.todoArr;
+  }
+
+  removeTask(item) {
+    let mySound = new Audio('sounds/failure.mp3')
```

Comment:
maybe you should consider load the audio once at the initializtion of the class and then only play it here instead of initializing new Audio each time